src/cfnlint/core.py

```diff
@@ -113,16 +127,19 @@ def create_parser():
         action='store_true', help='list all the rules'
     )
     parser.add_argument(
-        '-r', '--regions', dest='regions', default=['us-east-1'], nargs='*',
+        '-r', '--regions', dest='regions', nargs='+', default=[],
```

Comment:
I think we need to keep the default of 'us-east-1'  or have another way to load a default value.  We use that value to pick a spec file.